### PR TITLE
Add Examples repo reference to docs

### PR DESCRIPTION
**Pull request by aditya520**

### CODE REVIEW

File: pages/price-feeds/migrate-an-app-to-pyth/chainlink.md [hunk 1]
```diff
@@ -90,6 +90,8 @@ contract PythAggregatorV3Deployment is Script {
 
 ```
 
+Check the [Chainlink Migration Example](https://github.com/pyth-network/pyth-examples/tree/main/price_feeds/evm/chainlink_migration) for a complete example of how to deploy the `PythAggregatorV3` contract.
```
Comment (aditya520) [inline pages/price-feeds/migrate-an-app-to-pyth/chainlink.md:93]: I am not happy with the words "complete example". Can't think of any other at the moment.

Comment (jayantk) [reply pages/price-feeds/migrate-an-app-to-pyth/chainlink.md:93]: How about "for a runnable version of the example code above"?  

nitpick: I don't think "Check" is the right verb here. I think "Please see" is the best option, or possibly "Consult" or "Refer to".

File: pages/price-feeds/_meta.json [hunk 2]
```diff
@@ -42,8 +42,10 @@
   "error-codes": "Error Codes",
   "api-instances-and-providers": "API Instances and Providers",
   "contract-addresses": "Contract Addresses",
-  "examples": "Example Applications",
-
+  "examples": {
+    "title": "Examples ↗",
+    "href": "https://github.com/pyth-network/pyth-examples/tree/main/price_feeds"
+  },
```
Comment (jayantk) [inline pages/price-feeds/_meta.json:48]: I think we need an index of the available examples somewhere. I.e., we need something that lets people identify the examples that are relevant to them. Maybe that goes in the readme of the examples repo?

**Review by jayantk — APPROVED** [review]

File: pages/price-feeds/_meta.json (hunk 2 above)
Comment (jayantk) [reply pages/price-feeds/_meta.json:48]: err also I think this heading should be "Example Applications" not "Examples". The "Applications" makes it clear what you're going to find over there.

Comment (aditya520) [reply pages/price-feeds/_meta.json:48]: I will update the readme of the examples repo

**Pull request merged**